db/models.py

```diff
@@ -50,3 +53,63 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(on_delete=CASCADE, to="User")
```

Comment:
Code Style: It's better to use `get_user_model()` instead of `User`, it's a best practice. You can import it from `django.contrib.auth`.